File: 3kan.js
function func1 () {
    var kan = 2;
    console.log("将棋の藤井%d冠が%d冠を達成しました", kan++, kan);
}
function func2 () {
    var kan = 2;
    console.log("将棋の藤井%d冠が%d冠を達成しました", kan, ++kan);
}
func1 ();
func2 ();

File: 3kan.c
#include <stdio.h>

void func1 () {
  int kan = 2;
  // Bad implementation
  printf("将棋の藤井%d冠が%d冠を達成しました\n", kan++, kan);
}

void func2 () {
  int kan = 2;
  // Bad implementation
  printf("将棋の藤井%d冠が%d冠を達成しました\n", kan, ++kan);
}

int main (int argc, char ** argv) {
  func1();
  func2();
}
